### PR TITLE
Cleanup cli options for chia wallet coins {list,split,combine}

### PR DESCRIPTION
**Pull request by emlowe**
Attempt to cleanup CLI commands for chia wallet coins {list,split,combine}

Removed the majority of short-codes.
Simplified most long options
Options that are multiple=True take a single item and therefore the option should not be plural.

Needs testing

### CODE REVIEW

File: chia/cmds/coins.py [hunk 1]
```diff
@@ -64,20 +62,20 @@ def list_cmd(
...
     id: int,
     show_unconfirmed: bool,
-    min_coin_amount: str,
-    max_coin_amount: str,
-    excluded_coin_ids: Tuple[str],
-    excluded_coin_amounts: Tuple[int],
+    min_amount: str,
+    max_amount: str,
+    coins_to_exclude: Tuple[str],
+    amounts_to_exclude: Tuple[int],
```
Comment (altendky) [inline chia/cmds/coins.py:68]: `str`?  In XCH?  And is it really a `Tuple`?  Or a `List`.

File: chia/cmds/coin_funcs.py [hunk 2]
```diff
@@ -36,7 +36,7 @@ async def async_list(args: Dict[str, Any], wallet_client: WalletRpcClient, finge
         return
     final_min_coin_amount: uint64 = uint64(int(min_coin_amount * mojo_per_unit))
     final_max_coin_amount: uint64 = uint64(int(max_coin_amount * mojo_per_unit))
-    final_excluded_amounts: List[uint64] = [uint64(int(amount * mojo_per_unit)) for amount in excluded_amounts]
+    final_excluded_amounts: List[uint64] = [uint64(int(Decimal(amount) * mojo_per_unit)) for amount in excluded_amounts]
```
Comment (altendky) [inline chia/cmds/coin_funcs.py:39]: I don't know what we usually do, but in general I would rather this happen right at the CLI.  But, why only this one and not the other amounts?

File: chia/cmds/coins.py [hunk 3]
```diff
@@ -64,20 +62,20 @@ def list_cmd(
...
     address_prefix = selected_network_address_prefix(config)
     extra_params = {
         "id": id,
-        "max_coin_amount": max_coin_amount,
-        "min_coin_amount": min_coin_amount,
-        "excluded_amounts": excluded_coin_amounts,
-        "excluded_coin_ids": list(excluded_coin_ids),
+        "max_coin_amount": max_amount,
+        "min_coin_amount": min_amount,
+        "excluded_amounts": amounts_to_exclude,
```
Comment (altendky) [inline chia/cmds/coins.py:77]: This isn't actually going to the RPC, so could change the keys as well if you wanted.  (someday this indirection just goes away)

**Review by altendky — CHANGES_REQUESTED** [review]

File: chia/cmds/coin_funcs.py (hunk 2 above)
Comment (emlowe) [reply chia/cmds/coin_funcs.py:39]: The others happen a few lines up at the top of the function:
```
    min_coin_amount = Decimal(args["min_coin_amount"])
    max_coin_amount = Decimal(args["max_coin_amount"])
```

File: chia/cmds/coins.py (hunk 1 above)
Comment (emlowe) [reply chia/cmds/coins.py:68]: I believe it should be a List.

File: chia/cmds/coin_funcs.py (hunk 2 above)
Comment (altendky) [reply chia/cmds/coin_funcs.py:39]: I see, they are handled further up.  Probably good to do them all in one place.  `Decimal` conversion above and mojo conversion here.

File: chia/cmds/coins.py (hunk 1 above)
Comment (emlowe) [reply chia/cmds/coins.py]: Changes made Tuple->List. I believe they should all be Lists as changed

**Review by altendky — APPROVED** [review]

**Review by jack60612 — APPROVED** [review]
lgtm

**Review by paninaro — APPROVED** [review]

**Pull request merged**